tests/link/jax/test_nlinalg.py

```diff
@@ -82,7 +82,7 @@ def assert_fn(x, y):
 
 @pytest.mark.xfail(
     version_parse(jax.__version__) >= version_parse("0.2.12"),
-    reason="Omnistaging cannot be disabled",
+    reason="Operation leads to JAX Tracer object is used in a context where a Python integer is expected (jax.errors.TracerIntegerConversionError).",
 )
 def test_jax_basic_multiout_omni():
```

Comment:
Okay so the source of this error is just the dispatch implementation of Argmax using Jax arrays for the transpose axis and the reshape operation. Changing those to use numpy arrays and converting to tuple before they are actually used allows the test to pass. Something like this:

![image](https://github.com/pymc-devs/pytensor/assets/28983449/3e0748c6-bdfa-4d9e-8487-23bb678b2a40)

Reply:
We should rename the test to be just test `max_and_argmax`, it is no longer a multioutput Operation, so the old name doesn't make sense. Also can you confirm this is the only test we had for Max / Argmax in JAX?

Reply:
Yes that's the only test for Max/Argmax we have.